### PR TITLE
Fix height of filters

### LINKED ISSUE #803: Margin between filter selector and selected filters
There is some free space between selected filters. That is good. The filter selector should have the same space, but is running more or less into the filter instead. That's not good. There should be the same amount of space between all elements.

Screenshot from 2024-07-03 22-27-12

### LINKED ISSUE #804: Height difference between filter and filter selector
The filter selector and the selected filters should have the same height. 

Screenshot from 2024-07-03 22-26-55

### PR DESCRIPTION
**Pull request by dennis531**
Basically adds flexbox to elements and removes some useless `<span>`.

Close #804
Close #803

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -403,7 +402,9 @@ div.table-filter {
     }
 
     .ng-multi-value {
-      align-self: center;
+      display: flex;
```
Comment (lkiesow) [inline src/styles/main.scss:405]: I would also suggest adding a slight `gap` here:
```suggestion
      display: flex;
      gap: 4px;
```

File: src/styles/main.scss [hunk 2]
```diff
@@ -369,8 +370,6 @@ div.table-filter {
     display: flex;
```
Comment (lkiesow) [inline src/styles/main.scss:370]: The filters now run into each other:
Screenshot from 2024-07-12 18-08-32


I think some `gap` would make this better:
```suggestion
    display: flex;
    gap: 4px;
```

**Review by lkiesow — CHANGES_REQUESTED** [review]

File: src/styles/main.scss (hunk 2 above)
Comment (lkiesow) [reply src/styles/main.scss:370]: This is how the filters look with both `gap` applied:

Screenshot from 2024-07-12 18-19-40

**Comment by dennis531** [comment]
@lkiesow Thanks for these suggestions. It looks much better now.

**Comment by dennis531** [comment]
Close #803

**Pull request merged**